src/widget/wpixmapstore.h

```diff
@@ -1,35 +1,35 @@
 #pragma once
 
-#include <QPixmap>
 #include <QHash>
-#include <QSharedPointer>
-#include <QSvgRenderer>
 #include <QImage>
-#include <QScopedPointer>
 #include <QPainter>
+#include <QPixmap>
 #include <QRectF>
 #include <QString>
+#include <QSvgRenderer>
+#include <memory>
 
 #include "skin/legacy/imgsource.h"
 #include "skin/legacy/pixmapsource.h"
 #include "widget/paintable.h"
 
-
-typedef QSharedPointer<Paintable> PaintablePointer;
-typedef QWeakPointer<Paintable> WeakPaintablePointer;
+typedef std::shared_ptr<Paintable> PaintablePointer;
+typedef std::weak_ptr<Paintable> WeakPaintablePointer;
 
```

Comment:
nitpick, https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rt-using
```suggestion
using PaintablePointer = std::shared_ptr<Paintable> ;
using WeakPaintablePointer = std::weak_ptr<Paintable> ;
```